bin/firebase-import.js

```diff
@@ -188,8 +188,11 @@ function chunkInternal(ref, json, forceSplit) {
 function ChunkUploader(chunks) {
   this.next = 0;
   this.chunks = chunks;
-  this.bar = new ProgressBar('Importing [:bar] :percent (:current/:total)',
-    { width: 50, total: chunks.length, incomplete: ' ' });
+  if (process.stdout.isTTY) {
```

Comment:
I think this logic is backwards?  (If it's a TTY, we *do* want to show the progress bar.)